samples/Mvc/Mvc.Server/Controllers/AuthenticationController.cs

```diff
@@ -13,33 +15,53 @@ public class AuthenticationController : Controller {
             ViewBag.ReturnUrl = returnUrl;
 
             // Note: in a real world application, you'd probably prefer creating a specific view model.
-            return View("SignIn", Context.GetExternalProviders());
+            return View("SignIn", new AuthenticationModel() {AuthenticationDescriptions = Context.GetExternalProviders()});
```

Comment:
https://en.wiktionary.org/wiki/nitpick

Here, you should add a space after the first brace and before the second one.
Anyway, I'm not sure the model is really needed (see my other comment).